data/hai missions.txt

```diff
@@ -1053,3 +1053,245 @@ mission "Hai Honeymoon"
+					goto wrong
+				`	(Make up stories about Nanachi's experience.)`
+					goto lie
+			label safe
+			`	He seems a bit surprised by your tone, but responds positively. "Well, yes of course you look like a very experienced captain. Sorry if I sounded a bit rude before, I'm doing repairs, and have gotten really stressed, you know how it is. I'm sure you can handle it, just be careful."`
+				accept
+			label wrong
+			`	"She could be blasted to bits in an instant by an Unfettered fleet is what could go wrong, dying because of stupid recklesness!" He sighs and looks back at the crates, then takes a while looking around the deserted spaceport, as if trying to find some other captain, until he finally turns to Nanachi. "But I guess beggars can't be choosers. Fine, I'll let you take the cargo there, but be careful! And just so we're clear, you're only getting a small cut of the pay, I haven't been flying like a maniac from one end of Hai space to the other to get nothing!"`
+				accept
+			label lie
+			`	You go on about how Nanachi was the one who taught you how to fly a ship, and how famous she is amongst the Hai pilots, but as he looks at you with great disdain, he interrupts your stories. "Oh of course, an experienced captain that has been running around the spaceport for hours and can't even figure out the job board! I felt like I could maybe trust you to take care of her and the cargo, buy if you won't at least be honest with a fellow captain I can't put you in charge of a mission like this. Now get out of here, I have better things to do than waste my time listening to nonsense!" You and Nanachi continue to look for someone willing to give her a job, but after hours it's clear your best bet was with that man. Unable to find any other way to help her, you apologize and wish her the best of luck.`
+				decline
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog `	You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
+	on complete
+		conversation
+			`	Shortly after her ship lands, you meet up with Nanachi, who is doing her best to hide her excitement of being on another planet for the first time, gawking at some Hai military ships in particular. "So, how did you like your first time flying a ship?" you ask, to try and snap her out of it. "Oh um, I had a bit of trouble with the controls at first since they're a bit different from the ones in Hai ships, but I figured it out. I've always dreamt of being on other worlds, exploring and learning everything about them, I'm glad I'm finally able to do it." You help her get the crates out of the cargo hold and contact the client. Her share of the payment is small, as the client transfers most of the money to that merchant, but that doesn't make Nanachi any less excited about getting her first paycheck.`
```

Comment:
`for the first time`
So you had the shuttle, but never left Giverstone until getting a job? How many days of looking for a job had passed for Nanachi to gain a "bad reputation". I already said how I'm not too keen on the whole idea of Nanachi having a bad rep just for looking for a job, so perhaps clarify in the on offer conversation that Nanachi only recently, say either today or the day before, bought the ship.

Should be a new line after `snap her out of it.`, since it's a new person's dialog.

`a bit different from the ones in Hai ships`
So this is Nanachi's first ship, but she knows what Hai ship controls are like? Might want to elaborate on that either here or in the on offer conversation when she mentions buying the shuttle.

I'd put a new line after `I'm glad I'm finally able to do it."`

`that merchant`
I feel like you should say "the merchant she got the job from" or something similar, instead of just `that merchant`.

```suggestion
			`	Shortly after her ship lands, you meet up with Nanachi, who is doing her best to hide her excitement of being on another planet for the first time, gawking at some Hai military ships in particular. "So, how did you like your first time flying a ship?" you ask, to try and snap her out of it. "Oh um, I had a bit of trouble with the controls at first since they're a bit different from the ones in Hai ships, but I figured it out. I've always dreamt of being on other worlds, exploring and learning everything about them. I'm glad I'm finally able to do it." You help her get the crates out of the cargo hold and contact the client. Her share of the payment is small, as the client transfers most of the money to that merchant, but that doesn't make Nanachi any less excited about getting her first paycheck.`
```